daemon/logger/syslog/syslog.go

```diff
@@ -0,0 +1,55 @@
+package syslog
+
+import (
+	"log/syslog"
+	"os"
+	"path"
+
+	"github.com/docker/docker/daemon/logger"
+)
+
+type Syslog struct {
+	writer *syslog.Writer
+	tag    string
+}
+
+func New(tag string) (logger.Logger, error) {
+	return &Syslog{
+		writer: nil,
+		tag:    tag,
+	}, nil
+}
+
+func (s *Syslog) Log(msg *logger.Message) error {
+	if s.writer == nil {
+		log, err := syslog.New(syslog.LOG_USER, path.Base(os.Args[0]))
```

Comment:
If docker itself is logging, we could make it `syslog.LOG_DAEMON`, the logs will be from containers, therefore I set it `syslog.LOG_USER`